lib/ui/icons.dart

```diff
@@ -114,6 +114,8 @@ class FlutterIcons {
   static const Icon extractMethod =
       UrlIcon('/icons/preview/extract_method.png');
 
+
+  static const Icon greyProgr = UrlIcon('/icons/perf/GreyProgr@2x.png');
```

Comment:
remove the @2x.
This code adopts the IntelliJ convention that the @2x icon is automatically added on hidpi devices.